DayrSuryan/DSEthiop16.xml

```diff
@@ -116,39 +121,46 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </layout>
                      </layoutDesc>
                   </objectDesc>
-                  
+
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
-                        <seg type="script"/>
+                        <seg type="script">20th century</seg>
                         <seg type="ink">Black, red.</seg>
-                        <seg type="rubrication">ማርያም፡, parts of punctuation signs and numerals, structuring elements 
+                        <seg type="rubrication">ማርያም፡, parts of punctuation signs and numerals, structuring elements
                            (on <locus target="#4v"/> ጽዮን፡, ወኢይምስሎ<supplied reason="omitted" resp="DR">፡</supplied>, እስመ፡ ክቡር፡;
                            on <locus target="#5r"/> ተአምሪሃ፡).</seg>
+                           <date>20th century</date>
+                           <desc>Executed with care but poorly,
+                             by an unskilled hand.</desc>
                      </handNote>
                   </handDesc>
-                  
+
                   <additions>
                      <list>
                         <item xml:id="e1">
                            <locus target="#2r #3r #3v"/>
-                           <desc>Omitted words and characters have been added above the line.</desc>
-                        </item> 
+                           <desc>Omitted words and characters
+                              have been added above the line.</desc>
+                        </item>
                      </list>
                   </additions>
-                  
+
                   <bindingDesc>
                      <binding xml:id="binding" contemporary="unknown">
-                        <decoNote xml:id="b1">No binding.</decoNote>
+                        <decoNote xml:id="b1">No binding,
+                          no sewing stations.</decoNote>
                      </binding>
                   </bindingDesc>
-                  
+
                </physDesc>
-                  
+
                <history>
-                  <provenance>
-                  </provenance>
-               </history>
-               
+                              <origin>
+                                 <origDate notBefore="1920" notAfter="1960" evidence="internal">First half of the 20th century.</origDate>
```

Comment:
by "internal", I understand that a date is specified in the manuscript, which does not seem to be the case here?